portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+/* IDs */
```

Comment:
I like how you organized this by id/class/tagname but it seems like the recommendation is to organize it by section https://google.github.io/styleguide/htmlcssguide.html#Section_Comments

This isn't totally necessary for this since if this were a larger project they would probably have separate files for each section so just an FYI.